packages/console/src/notebook/Editor.tsx

```diff
@@ -112,6 +113,10 @@ class Editor extends Component<EditorProps, Record<string, never>> {
     this.editor.layout();
     MonacoUtils.removeConflictingKeybindings(this.editor);
 
+    monaco.languages.registerLinkProvider('plaintext', {
+      provideLinks: MonacoUtils.getProvideLinks(),
```

Comment:
Without the method curried, can just do:
```suggestion
      provideLinks: MonacoUtils.provideLinks,
```